config.json

```diff
@@ -2034,66 +2034,76 @@
       {
         "groupId": "io.grpc",
         "artifactId": "grpc-android",
-        "version": "1.57.2",
-        "nugetVersion": "1.57.2",
+        "version": "1.58.0",
+        "nugetVersion": "1.58.0",
         "nugetId": "Xamarin.Grpc.Android",
         "dependencyOnly": false,
         "templateSet": "grpc"
       },
       {
         "groupId": "io.grpc",
         "artifactId": "grpc-api",
-        "version": "1.57.2",
-        "nugetVersion": "1.57.2",
+        "version": "1.58.0",
+        "nugetVersion": "1.58.0",
         "nugetId": "Xamarin.Grpc.Api",
         "dependencyOnly": false,
         "templateSet": "grpc"
       },
       {
         "groupId": "io.grpc",
         "artifactId": "grpc-context",
-        "version": "1.57.2",
-        "nugetVersion": "1.57.2",
+        "version": "1.58.0",
+        "nugetVersion": "1.58.0",
         "nugetId": "Xamarin.Grpc.Context",
         "dependencyOnly": false,
         "templateSet": "grpc"
       },
       {
         "groupId": "io.grpc",
         "artifactId": "grpc-core",
-        "version": "1.57.2",
-        "nugetVersion": "1.57.2",
+        "version": "1.58.0",
+        "nugetVersion": "1.58.0",
         "nugetId": "Xamarin.Grpc.Core",
         "dependencyOnly": false,
+        "excludedRuntimeDependencies": "io.grpc.grpc-util",
```

Comment:
Context: https://github.com/grpc/grpc-java/issues/10576

This is unfortunate, and something our system doesn't really support.

What needs to happen:
- We compile "util" with a `<ProjectReference>` to "core".
- We remove the NuGet dependency this will create in "util" to "core".
- We add a NuGet dependency in "core" to "util".

This is likely something we do not want to spend resources on now.

I guess our choices are:
- Not bind new versions of `io.grpc`
- Use this exclusion and see if users report breakages.

If users' apps are crashing they will need to add a `<PackageReference>` to `Xamarin.Grpc.Util`.

Reply:
How about this:

1. break the cycle
    add `"excludedRuntimeDependencies"`
    ```json
      {
        "groupId": "io.grpc",
        "artifactId": "grpc-core",
        "version": "1.58.0",
        "nugetVersion": "1.58.0",
        "nugetId": "Xamarin.Grpc.Core",
        "dependencyOnly": false,
        "excludedRuntimeDependencies": "io.grpc.grpc-util",
        "templateSet": "grpc"
      },    
    ```
   or
    ```json
      {
        "groupId": "io.grpc",
        "artifactId": "grpc-util",
        "version": "1.58.0",
        "nugetVersion": "1.58.0",
        "nugetId": "Xamarin.Grpc.Core",
        "dependencyOnly": false,
        "excludedRuntimeDependencies": "io.grpc.grpc-core",
        "templateSet": "grpc"
      },    
    ```
    and 
2. add nuget in template
    
   https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20231028-updates-all/templates/grpc/Project.cshtml#L143-L148

Reply:
OK. 
reverted to 1.57.2.